mobile/src/main/java/ch/epfl/sdp/mobile/state/StatefulPrepareGameScreen.kt

```diff
@@ -55,16 +57,27 @@ fun StatefulPrepareGameScreen(
  */
 class SnapshotPrepareGameScreenState(
     val user: AuthenticatedUser,
+    selected: String?,
     override val opponents: List<ProfileAdapter>,
     val navigateToGame: (match: Match) -> Unit,
     val cancelClick: () -> Unit,
     val chessFacade: ChessFacade,
     val scope: CoroutineScope,
 ) : PrepareGameScreenState<ProfileAdapter> {
 
+  constructor(
+      user: AuthenticatedUser,
+      opponents: List<ProfileAdapter>,
+      navigateToGame: (match: Match) -> Unit,
+      cancelClick: () -> Unit,
+      chessFacade: ChessFacade,
+      scope: CoroutineScope,
+  ) : this(user, null, opponents, navigateToGame, cancelClick, chessFacade, scope)
+
```

Comment:
Question : is this `constructor` still used ?

```suggestion
```

Reply:
No I don't think